README.md

```diff
@@ -80,15 +80,98 @@ See below for more details.
 
 ## Test POCS
 
-Once you have setup your computer (above), the next step is to test your setup.
-This is easy to do using our built-in test suite. In a terminal, simply type:
+POCS comes with a testing suite that allows it to test that all of the software works and is installed corretly. Running the test suite by default will use simulators for all of the hardware and is meant to test that the software works correctly. Additionally, the testing suite can be run with various flags to test that attached hardware is working properly.
```

Comment:
c/corretly/correctly/.

Please add line breaks.

Please mention that testing with hardware enabled should be done with caution, as there will be no safety checks (i.e. unparking of a mount while it is raining).

Reply:
Added line breaks here but don't necessarily want to change whole document in this PR.

Added warning about hardware in hardware section below.